src/docGeneratorMain.ts

```diff
@@ -280,6 +280,14 @@ async function main() {
     "i18next",
     false
   )
+
+  console.log("Generate eslint-plugin-typescript-sort-keys description files")
+  await docGenerator.downloadDocs(
+      (pattern) =>
+          `${githubBaseUrl}/infctr/eslint-plugin-typescript-sort-keys/tree/master/docs/rules/${pattern}.md`,
+      "xss",
```

Comment:
I think this is wrong since [the plugin](https://github.com/infctr/eslint-plugin-typescript-sort-keys) requires you to add
```json
{
  "plugins": ["typescript-sort-keys"]
}
```

This means you need to change `"xss"` with `"typescript-sort-keys"` and check if it actually generated new md files in `docs/description/`, otherwise something went wrong.